src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/UserRole.orm.xml

```diff
@@ -3,5 +3,17 @@
                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                   xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
     <entity name="Sulu\Bundle\SecurityBundle\Entity\UserRole" table="se_user_roles">
```

Comment:
this is not configured as a `mapped-superclass` right now. do we need to change this?